policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/tuition/weighted_average_tuition/technical_institutes.yaml

```diff
@@ -1,6 +1,9 @@
 description: Arkansas defines the weighted average tuition for technical institutes at this amount under the post secondary education tuition deduction.
 values:
   2021-01-01: 780
+  2022-01-01: 780
+  2023-01-01: 780
```

Comment:
```suggestion
```

Reply:
We dont need to specify the values, if they haven't changed from the previous year